src/status_im2/contexts/quo_preview/inputs/title_input.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im2.contexts.quo-preview.inputs.title-input
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [react-native.blur :as blur]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "disabled?"
+    :key   :disabled?
+    :type  :boolean}
+   {:label "blur?"
+    :key   :blur?
+    :type  :boolean}
+   {:label   "Cursor Color"
+    :key     :color
+    :type    :select
+    :options (map (fn [color] (let [key (get color :name)] {:key key :value key}))
```

Comment:
One-liners can easily become unreadable in Lisps, because we lose track of the code (also data) structure. I don't know about you folks, but I find the code below more readable.

```clojure
(map (fn [color]
       (let [key (get color :name)]
         {:key key :value key}))
     (quo/picker-colors))
```

Reply:
@J-Son89 this may squeak by as a one-liner :)
```clojure
(map (fn [{:keys [name]}] {:key name :value name}) (quo/picker-colors))
```